### DISCUSSION TITLE
Anyone like to export and share their categories?

### DISCUSSION THREAD

**Discussion by rthorntn** [Part-DB/Part-DB-server#775] [Q&A]
Hi,

Just starting out, I primarily want to use parts-db for electronics component storage (a large percentage from Digikey) but I would also like to store other hobby "things", some example parent categories:

- Audio
- Computing
- Electronics
- Tools

Anyone doing anything like that, if so, could you share an export?

My thinking (which could be incorrect) is if I'm going to import a bunch of categories then I should leave my categories empty until then to avoid duplication?

Thoughts and ideas are also welcome!

Thanks.

**Comment by virtadpt** [comment]
Why export categories?  Just make the ones you need.

**Comment by CyberTeckyLLC** [comment]
> Hi,
> 
> Just starting out, I primarily want to use parts-db for electronics component storage (a large percentage from Digikey) but I would also like to store other hobby "things", some example parent categories:
> 
>     * Audio
> 
>     * Computing
> 
>     * Electronics
> 
>     * Tools
> 
> 
> Anyone doing anything like that, if so, could you share an export?
> 
> My thinking (which could be incorrect) is if I'm going to import a bunch of categories then I should leave my categories empty until then to avoid duplication?
> 
> Thoughts and ideas are also welcome!
> 
> Thanks.

Certainly!

This is a work in progress, and I'm open to suggestions as well...

+ ACTIVE COMPONENTS
  - Amplifiers
  - Batteries
  - Diodes / Rectifiers
  - Displays
  - ICs / SMDs
  - Light Emitting Diodes (LEDs)
  - Solar PVs / Cells
  + Transistors
     - Composite Transistors
     - Compound Transistors
     - Darington Transistors
     + MOSFETS
        - Field Effect Transistors (FETs)
        - Junction FET Transistors (JFETs)
     - Phototransistors
     - Thyristors
 
+ PASSIVE COMPONENTS
    - Antennas
    - Blowers / Fans
    - Cables
    - Capacitors
    - Coils / Inductors
    - Connectors & Terminals
    - Crystals
    - Detectors / Sensors
    - Fuses / Circuit Breakers
    - Piezoelectric
    - Resistors
    - Transducers

**Reply by CyberTeckyLLC** [reply]
Also, if you would like to add 'devices' such as tools and 'systems' such as computers, I would suggest a NEW categorization solution:

+ ELECTRONIC PARTS
   - ACTIVE COMPONENTS
   - PASSIVE COMPONENTS
  
+ OTHER (MISC.)
   - COMPUTERS
      - HP Laptop 123 Model abc
      - Dell Desktop 567 Model def
   - TOOLS
     - Makita Cordless Drill Model abc 
     - DeWalt 1/2 HP Circular Saw Model def

**Comment by rthorntn** [comment]
Thanks, I'll use your categories to build off of, I started off like this because the detail at the category level you actually put parts into was detailed enough in the "All Parts" table view, it took me a few tries:

Electronics
-- Components
---- Capacitors
------ Capacitors - Arrays (SMT)
------ Capacitors - Arrays (THT)
------ Capacitors - Ceramic (SMT)
------ Capacitors - Ceramic (THT)
------ Capacitors - Electrolytic (SMT)
------ Capacitors - Electrolytic (THT)

So in "All Parts" for an SMT Electrolytic I get:

EEE-HC1H1R0R | 1 µF 50 V Electrolytic | Capacitors - Electrolytic (SMT) |   | Panasonic |   | 10

**Reply by CyberTeckyLLC** [reply]
Nice!

I found it to be a long process on figuring out how to start myself.  More feedback the better, Thanks!  Also, After creating my categories and inputting my part inventory, I realized I didn't have to be too specific on electronic component parts, i.e., Capacitors - Arrays (SMT), etc., as long as I used a detailed part 'description' that was consistent.  Granted, knowing which parts posses a specific function and/or quality is definitely important, however, the 'Search' function is Part-DB is real good at showing a given inventory just by searching for part names such as, "Capacitors".  That being said, no mater which direction a user decides to go with, consistency and standardization of naming conventions of each part is certainly the best route to take.

**Reply by rthorntn** [reply]
I guess I wanted the Category sort under show "All Parts" to be useful, so I could look at it and see I had a "gap", E.g. I don't have a common SMT capacitor in there at all.  At one point my Category was SMT or THT and that will not do at all lol.

Thanks for contributing, I think it's good for the project to see what other people are doing, on a fresh install you start with no categories at all, if you create a bunch of categories and add thousands of parts before you spot a gremlin it could be a real pita to fix it.

**Comment by CyberTeckyLLC** [comment]
Here's another view of my setup as well.  

![Part-DB-AllParts-Example1](https://github.com/user-attachments/assets/9a1e529c-6038-4e7d-a160-20b5d4076a95)

**Reply by rthorntn** [reply]
Cool, at least 86 slots, wow, what are you using for storage?

**Reply by CyberTeckyLLC** [reply]
Actually, I use [3] AideTek containers, each of them containing 96 drawers.  Highly recommend for those looking to store their electronic components in small confined spaces!  See discussion [HERE](https://github.com/Part-DB/Part-DB-server/issues/779)
